src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -108,67 +105,108 @@
              :unread?   (not (:read notification))}
             (activity-buttons notification))]]])
 
-(defn notifications-list
+(defn filter-selector-read-toggle
   []
-  (let [notifications (<sub [:activity-center/notifications-per-read-status])]
-    [rn/flat-list {:data           notifications
-                   :key-fn         :id
-                   :on-end-reached #(>evt [:activity-center.notifications/fetch-next-page])
-                   :render-fn      render-notification}]))
-
-(defn filter-selector-read []
-  (let [unread-filter-enabled?  (<sub [:activity-center/status-filter-unread-enabled?])]
+  (let [unread-filter-enabled? (<sub [:activity-center/filter-status-unread-enabled?])]
     ;; TODO: Replace the button by a Filter Selector component once available for use.
     [button/button {:icon     true
                     :type     (if unread-filter-enabled? :primary :outline)
                     :size     32
-                    :on-press #(if unread-filter-enabled?
-                                 (>evt [:activity-center.notifications/fetch-first-page {:status-filter :read}])
-                                 (>evt [:activity-center.notifications/fetch-first-page {:status-filter :unread}]))}
-     :unread]))
+                    :on-press #(>evt [:activity-center.notifications/fetch-first-page
+                                      {:filter-status (if unread-filter-enabled?
+                                                        :read
+                                                        :unread)}])}
+     :main-icons2/unread]))
 
-(defn activity-center []
+;; TODO(2022-10-07): The empty state is still under design analysis, so we
+;; shouldn't even care about translations at this point. A placeholder box is
+;; used instead of an image.
+(defn empty-tab
+  []
+  [rn/view {:style {:align-items      :center
+                    :flex             1
+                    :justify-content  :center
+                    :padding-vertical 12}}
+   [rn/view {:style {:background-color colors/neutral-80
+                     :height           120
+                     :margin-bottom    20
+                     :width            120}}]
+   [text/text {:size   :paragraph-1
+               :style  {:padding-bottom 2}
+               :weight :semi-bold}
+    "No notifications"]
+   [text/text {:size :paragraph-2}
+    "Your notifications will be here"]])
```

Comment:
FYI: There's a comment on top of this function explaining why there are no translations. This is still in flux, and designers told me this will probably change.

```clojure
;; TODO(2022-10-07): The empty state is still under design analysis, so we
;; shouldn't even care about translations at this point. A placeholder box is
;; used instead of an image.
```

Reply:
oh sorry, missed